### DISCUSSION TITLE
Minimum Android and/or Hardware to run this?

### DISCUSSION THREAD

**Discussion by Ltek** [feelfreelinux/octo4a#165] [Q&A] [answered]
Is there an FAQ with what the minimum Hardware and Android requirements are for this?

thx for helping the noob!

**Comment by AZ4Zach** [comment, marked as answer]
Tom's 3D has an article/video where he tried a few different levels of hardware https://toms3d.org/2021/11/05/octoprint-on-android-with-octo4a/
I've run mine with a Moto G5 plus and a kindle fire 6th gen, and without a webcam stream it seems to run fine. With webcam however it becomes laggy and will crash frequently. I'm planning on upgrading to a Pixel 2 however which seems similar to his mid tier level  which had decent results and is decently cheap.

**Comment by junkjunker** [comment]
I've got the same question, because the [FAQ](https://github.com/feelfreelinux/octo4a/wiki/FAQ) states:
"It is recommended to use an Android device with at least Android 5.0 (lollipop) for best results. Octo4a also comes with experimental support for older Androids up to 4.3."

"Up to 4.3" - does that mean Android 1 to 4.3, skip 4.4, and pick back up again at 5.0?

Or does it really mean Experimental Mode for Android 4.3 to 4.4, and regular for 5.0+?

Also, how does one find this Experimental Mode?

Thanks!
